fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/ContainmentTriplesServiceImpl.java

```diff
@@ -44,8 +44,9 @@
 
     @Override
     public Stream<Triple> get(final Transaction tx, final FedoraResource resource) {
-        final Node currentNode = createURI(resource.getFedoraId().getFullId());
-        return containmentIndex.getContains(txId(tx), resource).map(c ->
+        final var fedoraId = resource.getFedoraId();
+        final Node currentNode = createURI(fedoraId.getFullId());
```

Comment:
I think this can be removed, I'm just reading code on my phone so I could be mistaken.

Reply:
I think I need to keep both, currentNode is used on line 50, and fedoraId is used on 49. But thank you for the phone dedication

Reply:
🤦  I looked at these 5 lines for like 5 minutes and still couldn't get them correct in my mind. You are correct, please disregard my previous comment.